src/backend/core/api/viewsets.py

```diff
@@ -244,6 +245,9 @@ class TeamViewSet(
 
     permission_classes = [permissions.AccessPermission]
     serializer_class = serializers.TeamSerializer
+    filter_backends = [filters.OrderingFilter]
+    ordering_fields = ["created_at"]
```

Comment:
Nice one! Do you think we could also provide the option of ordering by name ?

Reply:
Yes you can (https://www.django-rest-framework.org/api-guide/filtering/#specifying-which-fields-may-be-ordered-against), but I don't need it in this PR, let's add it when we will need it.